README.md

```diff
@@ -102,6 +102,26 @@ _(Note that in previous versions, the `mysqlserver` environment variable used at
 and the `mysqlserver` environment variable is now no longer supported.)_
 
 
+Instructions to run using minikube and kubectl
+===================================================
+
+You can also run Fineract using containers on a kubernetes cluster.
+
+As Prerequisites, you must have `minikube` and `kubectl` installed on your machine; see
+[Minikube & Kubectl install](https://kubernetes.io/docs/tasks/tools/install-minikube/).
+
+Now to run a new Fineract instance on minikube you can simply:
+
+1. `git clone https://github.com/apache/fineract.git ; cd fineract`
+1. `minikube start`
+1. `kubectl apply -f kubernetes/`
```

Comment:
you don't want to deploy all simultaneously.@xurror You would want the MySQL server up and running before starting up the fineract server. Unfortunately, the isn't a way to specify that with Kubernetes to the best of my knowledge. But basically you shouldn't start them up simultaneously.